src/main/resources/templates/fantasy/index.html

```
@@ -48,13 +48,17 @@ <h3>Rules & Scoring</h3>
                         </div>
                     </div>
                     <p style="margin-top: 10px; color: #666; font-style: italic;">
-                        <br> 1. ㄹ 자 드래프트로 18인 픽 (타자 9 / 투수 9) </br>
+                        <br> 1. 드래프트룰 RULE1 : 18인 드래프트 (타자 9/ 투수 9) </br>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </br> ]

(tag-pair)